spec/factories/date_ranges.rb

```diff
@@ -28,5 +28,11 @@
             start_date Date.tomorrow
             end_date Date.tomorrow
         end
+
+        trait :force_past do
```

Comment:
I'm not completely sure about RoR naming conditions, but I think "Entity :withX :withoutY" etc reads better than these titles.